packages/cli/src/runners/task-broker.service.ts

```diff
@@ -90,35 +87,35 @@ export class TaskBroker {
 	}
 
 	registerRunner(runner: TaskRunner, messageCallback: MessageCallback) {
-		this.knownRunners[runner.id] = { runner, messageCallback };
+		this.knownRunners.set(runner.id, { runner, messageCallback });
 	}
 
 	deregisterRunner(runnerId: string) {
-		if (runnerId in this.knownRunners) {
-			delete this.knownRunners[runnerId];
+		if (this.knownRunners.has(runnerId)) {
+			this.knownRunners.delete(runnerId);
 		}
```

Comment:
nit: can also delete directly, as it's no-op if the key is not in the map

```suggestion
	this.knownRunners.delete(runnerId);
```